### DISCUSSION TITLE
Signed nuget package

### DISCUSSION THREAD

**Discussion by tisonv** [serilog-contrib/serilog-sinks-grafana-loki#188] [Ideas]
Hi !

Would you consider signing the nuget package or releasing a second _.signed_ one ?

Thanks !

**Comment by mishamyte** [comment]
Hi @tisonv!

Nice suggestion. Lemme think how it will be better to do and investigate possibility of pipelining that stuff

**Comment by tisonv** [comment]
Hello !

Though I learnt afterwards that signing is not that useful anymore if the project targets > .Net Core 5
https://learn.microsoft.com/en-us/dotnet/standard/assembly/strong-named

Serilog targets .Net Standard 2.0 and is signed so it might be a good idea to do the same

Just add a snk to the project and add it to each project
https://learn.microsoft.com/en-us/dotnet/standard/assembly/sign-strong-name

**Reply by mishamyte** [reply]
I think it's implemented in #215 and sorry for the late answer
